src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -61,6 +61,37 @@ public static bool IsZipDeployment(this IEnvironment environment)
                 !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage));
         }
 
+        public static bool IsAppServiceWindowsEnvironment(this IEnvironment environment)
+        {
+            return environment.IsAppServiceEnvironment()
+                && !environment.IsLinuxAppServiceEnvironment()
+                && !environment.IsLinuxContainerEnvironment();
+        }
+
+        public static bool IsCoreToolsEnvironment(this IEnvironment environment)
```

Comment:
What does coretools environment mean? You're running locally using core tools? Or the app was created with core tools? If the latter, then rather than this new env var, I think you should just check that the required storage is available.

Reply:
IsCoreToolsEnvironment means that the app is running locally using core tools. I should may be rename the method `IsRunningOnCoreTools` or something more accurate

Reply:
Are you injecting a new environment variable in core tools? I agree that something like `IsCoreToolsHosted` would be a better name than environment (applies to environment variable name as well)